oauth2client/contrib/gce.py

```diff
@@ -84,7 +84,8 @@ def _refresh(self, http_request):
         """
         query = '?scope=%s' % urllib.parse.quote(self.scope, '')
         uri = META.replace('{?scope}', query)
-        response, content = http_request(uri)
+        headers = { 'Metadata-Flavor': 'Google' }
+        response, content = http_request(uri, headers=headers)
```

Comment:
Drop the introduction of the local field `headers` and in general be reticent about introducing a new name only to use it exactly once and on the immediately following line. Just use the dictionary literal expression in this call.

If `http_request(uri, headers={'Metadata-Flavor': 'Google'})` were a too-complex expression that spread across three lines, defining a named value for some intermediate part of it would be fine, but that's not the case here.